src/utils/kit-utils.js

```diff
@@ -58,7 +58,7 @@ export const readFile = (reader, file) => {
 export function dataURLtoFile(dataurl, filename) {
   const arr = dataurl.split(',')
   const mime = arr[0].match(/:(.*?);/)[1]
-  const bstr = atob(arr[1])
+  const bstr = window.atob(arr[1])
```

Comment:
Its show `atob` like deprecated method, but the nodejs version its deprecated not the DOM one.

So using window solves and avoid possible future bugs.